api/stats.proto

```diff
@@ -54,73 +54,84 @@ message StatsConfig {
   google.protobuf.BoolValue use_all_default_tags = 2;
 }
 
-// Designates a tag to strip from the tag extracted name and provide as a named
-// tag value for all statistics. This will only occur if any part of the name
-// matches the regex provided with one or more capture groups.
+// Configuration for tagged metrics. Setting default tags which have fixed tag
+// value and will be added to all metrics. Or settting dynamic tag extractions
+// with regex.
 message TagSpecifier {
   // Attaches an identifier to the tag values to identify the tag being in the
   // sink. Envoy has a set of default names and regexes to extract dynamic
   // portions of existing stats, which can be found in `well_known_names.h
   // <https://github.com/envoyproxy/envoy/blob/master/source/common/config/well_known_names.h>`_
   // in the Envoy repository. If a :ref:`tag_name
-  // <envoy_api_field_TagSpecifier.tag_name>` is provided in the config with an
-  // empty regex, Envoy will attempt to find that name in its set of defaults
-  // and use the accompanying regex.
+  // <envoy_api_field_TagSpecifier.tag_name>` is provided in the config and none of
+  // :ref:`regex <envoy_api_field_TagSpecifier.regex>` and
+  // :ref:`fixed_value <envoy_api_field_TagSpecifier.fixed_value>` were specified,
+  // Envoy will attempt to find that name in its set of defaults and use the accompanying regex.
   //
   // .. note::
   //
-  //   If any default tags are specified twice, the config will be considered
+  //   If any tag names are specified twice, the config will be considered
```

Comment:
The comment can be simplified as

// It is invalid to specify the same tag name twice in a config.